src/aap_eda/api/views/activation.py

```diff
@@ -188,7 +190,7 @@ def partial_update(self, request, pk):
         )
 
         if is_enabled:
-            response = self._start(activation)
+            response = self._start(request, activation)
```

Comment:
why not sending request_id to `_start()`?

Reply:
keep function `enable` simple